<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>738</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正悠黑_510M</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid rgb(50, 50, 50);
background: rgb(50, 50, 50);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: rgb(50, 50, 50);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainLayout">
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
        font-size: 30px;
        border: 1px solid rgb(80, 80, 80);
        border-radius: 4px;
        margin-top: 10px;
}
QGroupBox::title {
        color: rgb(227, 234, 242);
        top: -12px;
        left: 10px;
}</string>
          </property>
          <property name="title">
           <string>信息摘要</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>70</y>
             <width>72</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(225, 225, 225);
font: 63 12pt &quot;方正悠黑_510M&quot;;</string>
           </property>
           <property name="text">
            <string>人数：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>120</y>
             <width>72</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(225, 225, 225);
font: 63 12pt &quot;方正悠黑_510M&quot;;</string>
           </property>
           <property name="text">
            <string>速率：</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#widget_2 {
        border: 1px solid rgb(80, 80, 80);
border-radius: 4px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="summaryButton">
          <property name="font">
           <font>
            <family>方正悠黑_510M</family>
            <pointsize>12</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 25px;
font: 63 12pt &quot;方正悠黑_510M&quot;;
}
QPushButton:enabled {
        background: rgb(68, 69, 73);
        color: rgb(227, 234, 242);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(227, 234, 242);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
</string>
          </property>
          <property name="text">
           <string>视频摘要</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pedestrianButton">
          <property name="font">
           <font>
            <family>方正悠黑_510M</family>
            <pointsize>12</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 25px;
font: 63 12pt &quot;方正悠黑_510M&quot;;
}
QPushButton:enabled {
        background: rgb(68, 69, 73);
        color: rgb(227, 234, 242);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(227, 234, 242);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
</string>
          </property>
          <property name="text">
           <string>行人检测</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="flowButton">
          <property name="font">
           <font>
            <family>方正悠黑_510M</family>
            <pointsize>12</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 25px;
font: 63 12pt &quot;方正悠黑_510M&quot;;
}
QPushButton:enabled {
        background: rgb(68, 69, 73);
        color: rgb(227, 234, 242);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(227, 234, 242);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
</string>
          </property>
          <property name="text">
           <string>流量统计</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="vehicleButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>方正悠黑_510M</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 25px;
}
QPushButton:enabled {
        background: rgb(68, 69, 73);
        color: rgb(227, 234, 242);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(227, 234, 242);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
	font: 63 12pt &quot;方正悠黑_510M&quot;;</string>
          </property>
          <property name="text">
           <string>车辆检测</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="7">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
        font-size: 30px;
        border: 1px solid rgb(80, 80, 80);
        border-radius: 4px;
        margin-top: 10px;
}
QGroupBox::title {
        color: rgb(227, 234, 242);
        top: -12px;
        left: 10px;
}</string>
          </property>
          <property name="title">
           <string>视频信息</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>40</y>
             <width>261</width>
             <height>271</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(225, 225, 225);
font: 63 12pt &quot;方正悠黑_510M&quot;;</string>
              </property>
              <property name="text">
               <string>拍摄时间</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="videoTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(225, 225, 225);</string>
              </property>
              <property name="text">
               <string>无数据</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="videoLength">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(225, 225, 225);</string>
              </property>
              <property name="text">
               <string>无数据</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(225, 225, 225);
font: 63 12pt &quot;方正悠黑_510M&quot;;</string>
              </property>
              <property name="text">
               <string>视频名称</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="videoName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(225, 225, 225);</string>
              </property>
              <property name="text">
               <string>无数据</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(225, 225, 225);
font: 63 12pt &quot;方正悠黑_510M&quot;;</string>
              </property>
              <property name="text">
               <string>视频时长</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="3" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="playButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 25px;
	font: 63 9pt &quot;方正悠黑_510M&quot;;
}
QPushButton:enabled {
        background: rgb(68, 69, 73);
        color: rgb(227, 234, 242);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(227, 234, 242);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
</string>
          </property>
          <property name="text">
           <string>播放/暂停</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="videoLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>无数据</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="fullButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
        border-radius: 4px;
        border: none;
        width: 75px;
        height: 25px;
	font: 63 9pt &quot;方正悠黑_510M&quot;;
}
QPushButton:enabled {
        background: rgb(68, 69, 73);
        color: rgb(227, 234, 242);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(227, 234, 242);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
</string>
          </property>
          <property name="text">
           <string>全屏</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="horizontalSlider">
          <property name="styleSheet">
           <string notr="true">QSlider:horizontal {
        height: 20px;
        background: transparent;
        margin-top: 3px;
        margin-bottom: 3px;
}
QSlider::handle:horizontal {
        height: 20px;
        min-width: 30px;
        background: rgb(68, 69, 73);
        margin-left: 15px;
        margin-right: 15px;
}
QSlider::handle:horizontal:hover {
        background: rgb(80, 80, 80);
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
        background: rgb(57, 58, 60);
        border: none;
}
QMenuBar::item {
        padding: 5px 10px 5px 10px;
        background: transparent;
}
QMenuBar::item:enabled {
        color: rgb(227, 234, 242);
}
QMenuBar::item:!enabled {
        color: rgb(155, 155, 155);
}
QMenuBar::item:enabled:selected {
        background: rgba(255, 255, 255, 40);
}</string>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="styleSheet">
     <string notr="true">QMenu {
        border: 1px solid rgb(100, 100, 100);
        background: rgb(68, 69, 73);
}
QMenu::item {
        height: 22px;
        padding: 0px 25px 0px 20px;
}
QMenu::item:enabled {
        color: rgb(225, 225, 225);
}
QMenu::item:!enabled {
        color: rgb(155, 155, 155);
}
QMenu::item:enabled:selected {
        color: rgb(230, 230, 230);
        background: rgba(255, 255, 255, 40);
}
QMenu::separator {
        height: 1px;
        background: rgb(100, 100, 100);
}
QMenu::indicator {
        width: 13px;
        height: 13px;
}
QMenu::icon {
        padding-left: 2px;
        padding-right: 2px;
}</string>
    </property>
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="styleSheet">
     <string notr="true">QMenu {
        border: 1px solid rgb(100, 100, 100);
        background: rgb(68, 69, 73);
}
QMenu::item {
        height: 22px;
        padding: 0px 25px 0px 20px;
}
QMenu::item:enabled {
        color: rgb(225, 225, 225);
}
QMenu::item:!enabled {
        color: rgb(155, 155, 155);
}
QMenu::item:enabled:selected {
        color: rgb(230, 230, 230);
        background: rgba(255, 255, 255, 40);
}
QMenu::separator {
        height: 1px;
        background: rgb(100, 100, 100);
}
QMenu::indicator {
        width: 13px;
        height: 13px;
}
QMenu::icon {
        padding-left: 2px;
        padding-right: 2px;
}</string>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionview"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="styleSheet">
     <string notr="true">QMenu {
        border: 1px solid rgb(100, 100, 100);
        background: rgb(68, 69, 73);
}
QMenu::item {
        height: 22px;
        padding: 0px 25px 0px 20px;
}
QMenu::item:enabled {
        color: rgb(225, 225, 225);
}
QMenu::item:!enabled {
        color: rgb(155, 155, 155);
}
QMenu::item:enabled:selected {
        color: rgb(230, 230, 230);
        background: rgba(255, 255, 255, 40);
}
QMenu::separator {
        height: 1px;
        background: rgb(100, 100, 100);
}
QMenu::indicator {
        width: 13px;
        height: 13px;
}
QMenu::icon {
        padding-left: 2px;
        padding-right: 2px;
}</string>
    </property>
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionwindows"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">QMenu {
        border: 1px solid rgb(100, 100, 100);
        background: rgb(68, 69, 73);
}
QMenu::item {
        height: 22px;
        padding: 0px 25px 0px 20px;
}
QMenu::item:enabled {
        color: rgb(225, 225, 225);
}
QMenu::item:!enabled {
        color: rgb(155, 155, 155);
}
QMenu::item:enabled:selected {
        color: rgb(230, 230, 230);
        background: rgba(255, 255, 255, 40);
}
QMenu::separator {
        height: 1px;
        background: rgb(100, 100, 100);
}
QMenu::indicator {
        width: 13px;
        height: 13px;
}
QMenu::icon {
        padding-left: 2px;
        padding-right: 2px;
}</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actioncontent"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="styleSheet">
     <string notr="true">QMenu {
        border: 1px solid rgb(100, 100, 100);
        background: rgb(68, 69, 73);
}
QMenu::item {
        height: 22px;
        padding: 0px 25px 0px 20px;
}
QMenu::item:enabled {
        color: rgb(225, 225, 225);
}
QMenu::item:!enabled {
        color: rgb(155, 155, 155);
}
QMenu::item:enabled:selected {
        color: rgb(230, 230, 230);
        background: rgba(255, 255, 255, 40);
}
QMenu::separator {
        height: 1px;
        background: rgb(100, 100, 100);
}
QMenu::indicator {
        width: 13px;
        height: 13px;
}
QMenu::icon {
        padding-left: 2px;
        padding-right: 2px;
}</string>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="aboutUs"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuView"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background: rgb(57, 58, 60);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="aboutUs">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionview">
   <property name="text">
    <string>view</string>
   </property>
  </action>
  <action name="actionwindows">
   <property name="text">
    <string>windows</string>
   </property>
  </action>
  <action name="actioncontent">
   <property name="text">
    <string>content</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>最近文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
